app/controllers/bundle_context_controller.rb

```diff
@@ -1,11 +1,26 @@
 class BundleContextController < ApplicationController
-
   def index
+    @bundle_context = BundleContext.new
   end
 
   def create
-    logger.debug("BundleContextController.create called")
-    PreassemblyJob.perform_later
+    @bundle_context = BundleContext.new(bundle_context_params)
+
+    # TODO: ticket #270 user should be authenticated as saved in AR
+    user = User.create(sunet_id: 'temp')
+
+    @bundle_context.user = user
+    if @bundle_context.save
+      # TODO: ticket #267 Choose type of job (DiscoveryReport or Pre-Assembly) using params[:job_selection]
+      PreassemblyJob.perform_later
```

Comment:
This doesn't mesh w/ the approach I take in #281, mainly because:
1. the job will take (at least) one argument, a `JobRun` instance that will be used to record output and link back to this `@bundle_context`
2. It won't be just one job class being invoked
3. in #281, the `perform_later` is hooked in by default after creating a `JobRun`, so the controller is off the hook for knowing which kind of job to call (and how).

I can clean this up after a merge, but I wanted to mention it at least.